src/App.jsx

```diff
@@ -0,0 +1,62 @@
+/* eslint linebreak-style: ["error", "windows"] */
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+} from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import Navbar from './components/navbar/Navbar';
+import AboutScreen from './routes/About';
+import AdminScreen from './routes/admin/Admin';
+import BookingScreen from './routes/Booking';
+import DetailScreen from './routes/Detail';
+import HomeScreen from './routes/Home';
+import NewScreen from './routes/admin/New';
+import ReservationScreen from './routes/Reservation';
+import LoginScreen from './routes/Login';
+import SignupScreen from './routes/Signup';
+import ProtectedRoutes from './routes/ProtectedRoutes';
+import { getVehicles } from './redux/vehicles/vehicles';
+import { getReservations } from './redux/users/users';
+
+function App() {

```

Comment:
- [ ] Almost all over your frontend project, you are using the `function` keyword.  Please, kindly refactor all your functions to use ES6 arrow functions.

## Explanation

All over the software industry, the use of the `function` keyword has been soft deprecated.  Most likely, your job will require ES6 .

## Tips and extra help

Just search for the keyword on VsCode (ignore the snapshots and package.lock.json):

![image](https://user-images.githubusercontent.com/57421823/198726201-0ee2d8d4-6d5e-4946-bc80-7005eefea415.png)

Just focus on your components, redux, and other helper functions that you wrote.


That way, your app will comply with the latest industry standards 💪